smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/expects-br.smithy

```diff
@@ -0,0 +1,11 @@
+$version: "2" /// Ignore me
+
+namespace com.example /// Doc comment on Foo
+
+string Foo /// Doc comment on Bar
+
+string Bar /// Doc comment on Baz
+
+string Baz, /// Doc comment on Bam
+
+string Bam,,,/// Dangling
```

Comment:
This example looks to me like a good reason for not making this change. If I were writing a doc comment like this, I would expected it to apply to the shape that it shares a line with

Reply:
Totally agree. I don't like it at all. It worked before though, and it's allowed in the grammar. I updated the PR to emit a warning for this. I think we should consider removing support in the future after the warning has had time to be seen by modelers. I did see some impacted models inside of Amazon.